tests/paddle/test_paddle_autolog.py

```diff
@@ -89,8 +89,8 @@ def test_autolog_log_models_configuration(log_models):
     with mlflow.start_run() as run:
         train_model()
 
-    artifacts = MlflowClient().list_artifacts(run.info.run_id)
-    assert any(x.path == "model" for x in artifacts) == log_models
+    MlflowClient().list_artifacts(run.info.run_id)
+    assert (mlflow.last_logged_model() is not None) == log_models
```

Comment:
Will rebase this branch once https://github.com/mlflow/mlflow/pull/14694 is merged